apps/web/src/ee/clerk/providers/ClerkProvider.tsx

```diff
@@ -5,54 +5,103 @@ import { ClerkProp, ClerkProvider as _ClerkProvider } from '@clerk/clerk-react';
-    width: 'var(--nv-sizes-125)',
-  },
-};
-
-const localization = {
+const CLERK_LOCALIZATION = {
   userProfile: {
     navbar: {
       title: 'Settings',
       description: '',
+      account: 'User profile',
+      security: 'Access security',
     },
   },
+
+  userButton: {
+    action__signOut: 'Log Out',
+  },
 };
 
 const ALLOWED_REDIRECT_ORIGINS = ['http://localhost:*', location.origin];
 
 export const ClerkProvider: React.FC<PropsWithChildren<{}>> = ({ children }) => {
+  const { colorScheme } = useColorScheme();
+
+  const CLERK_MODAL_ELEMENT = {
+    modalContent: {
+      width: '80rem',
+      display: 'block',
+    },
+    modalBackdrop: {
+      zIndex: 200,
+    },
+    cardBox: {
+      width: '100%',
+    },
+    rootBox: {
+      width: 'auto',
+    },
+    navbar: {
+      background: 'var(--nv-colors-surface-panel) !important',
+      '& h1': {
+        fontFamily: 'var(--nv-fonts-system) !important',
+        fontWeight: 'var(--nv-font-weights-strong) !important',
+        fontSize: 'var(--nv-font-sizes-150) !important',
+        letterSpacing: '0',
+        textDecoration: 'none',
+        lineHeight: 'var(--nv-line-heights-200) !important',
+      },
+    },
+    headerTitle: {
+      fontFamily: 'var(--nv-fonts-system) !important',
+      fontWeight: 'var(--nv-font-weights-strong) !important',
+      fontSize: 'var(--nv-font-sizes-125) !important',
+      letterSpacing: '0',
+      textDecoration: 'none',
+      lineHeight: 'var(--nv-line-heights-175) !important',
+    },
+    navbarButton: {
+      fontFamily: 'var(--nv-fonts-system) !important',
+      fontWeight: 'var(--nv-font-weights-strong) !important',
+      fontSize: 'var(--nv-font-sizes-88) !important',
+      paddingTop: 'var(--nv-spacing-50) !important',
+      paddingBottom: 'var(--nv-spacing-50) !important',
+      color: 'var(--nv-colors-typography-text-secondary) !important',
+      '&.cl-active': {
+        color: 'var(--nv-colors-typography-text-main) !important',
+        background: 'var(--nv-colors-surface-page) !important',
+        '& svg': {
+          color: 'var(--nv-colors-typography-text-main) !important',
+          fill: 'var(--nv-colors-typography-text-main) !important',
+          opacity: '1',
+        },
+      },
+      '&:hover, &:focus, &:active': {
+        color: 'var(--nv-colors-typography-text-main) !important',
+        background: 'var(--nv-colors-surface-page) !important',
+        opacity: '1',
+        '& svg': {
+          color: 'var(--nv-colors-typography-text-main) !important',
+          fill: 'var(--nv-colors-typography-text-main) !important',
+          opacity: '1',
+        },
+      },
+    },
+    navbarButtonIcon: {
+      height: 'var(--nv-sizes-125)',
+      width: 'var(--nv-sizes-125)',
+      opacity: '1 !important',
+    },
+  };
+
+  const CLERK_OVERRIDE_VARIABLES: Variables = {
+    fontFamily: 'var(--nv-fonts-system)',
+    fontSize: 'var(--nv-font-sizes-88)',
+  };
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Move outside the component. Can even move to a `ClerkProvider.styles.ts` (or `ClerkProvider.const.ts`) file, but we don't want these redefined on every render.‏

Reply:
done